src/app/reports/invoice/invoice.html

```diff
@@ -16,13 +16,13 @@ <h3 class="panel-title">Delivery Rounds</h3>
       </tr>
     </thead>
     <tbody>
-      <tr ng-repeat="round in reportsCtrl.deliveryRounds">
+      <tr ng-repeat="round in reportsAllCtrl.deliveryRounds">
```

Comment:
actually this page will likely change before next deployment, have already discussed with Janet about pagination and other optimization.